guides/common/modules/con_applying-custom-configuration.adoc

```diff
@@ -4,7 +4,7 @@
 When you install and configure {Project} for the first time using `{foreman-installer}`, you can specify that the DNS and DHCP configuration files are not to be managed by Puppet using the installer flags `--foreman-proxy-dns-managed=false` and `--foreman-proxy-dhcp-managed=false`.
 If these flags are not specified during the initial installer run, rerunning of the installer overwrites all manual changes, for example, rerun for upgrade purposes.
 If changes are overwritten, you must run the restore procedure to restore the manual changes.
-For more information, see {InstallingProjectDocURL}restoring-manual-changes-overwritten-by-a-puppet-run_{project-context}[Restoring Manual Changes Overwritten by a Puppet Run].
+For more information, see {InstallingProjectDocURL}restoring-manual-changes-overwritten-by-a-puppet-run_{project-context}[Restoring Manual Changes Overwritten by a Puppet Run] in _{InstallingServerDocTitle}_.
```

Comment:
This looks correct to me, but I wish we'd reword "InstallingProjectDocURL" to "InstallingServerDocURL" later in a separate PR.

Reply:
Totally agreed.